content/posts/newsletter/2021-02-05.md

```diff
@@ -0,0 +1,240 @@
+    -   This document defines an XMPP protocol extension that enables a server to communicate issues with the server to all users in a semantic manner.
+
+### New
+
+-   Version 0.1.0 of [XEP-0454](https://xmpp.org/extensions/xep-0454.html) (OMEMO Media sharing)
+    -   An informal way of sharing media files despite limitations in the OMEMO encryption
+    -   Accepted by vote of Council on 2021-01-13.
+
+-   Version 0.1.0 of [XEP-0453](https://xmpp.org/extensions/xep-0453.html) (DOAP usage in XMPP)
+    -   This specification defines how XMPP projects can provide a machine-readable description of their abilities, and how external entities can interact with it.
+    -   Accepted by vote of Council on 2021-01-13.
+
+-   Version 0.1.0 of [XEP-0452](https://xmpp.org/extensions/xep-0452.html) (MUC Mention Notifications)
+    -   This specification documents how a user may be informed when they're mentioned in a MUC which they're not currently joined to.
+    -   Accepted by vote of Council on 2021-01-06.
+
+-   Version 0.1.0 of [XEP-0451](https://xmpp.org/extensions/xep-0451.html) (Stanza Multiplexing)
+    -   This spec provides a mechanism for multiplexing multiple virtual hosts over a single XMPP session.
+    -   Accepted by vote of Council on 2020-12-09.
+
+-   Version 0.1.0 of [XEP-0450](https://xmpp.org/extensions/xep-0450.html) (Automatic Trust Management (ATM))
+    -   This document specifies a way to automatically manage the trust in public long-term keys used by end-to-end encryption protocols.
+    -   Accepted by vote of Council on 2020-12-02.
+
+### Deferred
+
+If an experimental XEP isn’t updated for more than six months, it will be moved off Experimental to Deferred. If there is another update, it will put the XEP back onto Experimental.
+
+-   No XEPs deferred this month.
+
+### Updated
+
+-   Version 1.1.0 of [XEP-0393](https://xmpp.org/extensions/xep-0393.html) (Message Styling)
+    -   Clarify parsing rules for span directives (sw)
+
+-   Version 1.1 of [XEP-0176](https://xmpp.org/extensions/xep-0176.html) (Jingle ICE-UDP Transport Method)
+    -   Make the 'foundation' attribute a string instead of an unsignedByte.
+    -   Make the 'network' attribute optional, and add a mapping to SDP. (egp)
+
+-   Version 0.4.0 and 0.5.0 of [XEP-0372](https://xmpp.org/extensions/xep-0372.html) (References)
+    -   Specify counting should be of code points. (kis)
+    -   Specify that begin is inclusive, starts counting at zero, and that end is exclusive (Dijkstra-based convention) (gh/jcbrand)
+
+-   Version 0.2.0 and 0.3.0 of [XEP-0434](https://xmpp.org/extensions/xep-0434.html) (Trust Messages (TM))
+    -   Clarify usage of trust messages by protocols such as Automatic Trust Management (ATM) ([XEP-0450](https://xmpp.org/extensions/xep-0450.html))
+    -   Use namespace 'urn:xmpp:atm:0' of Automatic Trust Management (ATM) ([XEP-0450](https://xmpp.org/extensions/xep-0450.html)) as example for 'usage' attribute.
```

Comment:
Might need to wrap `urn:xmpp:atm:0` in backquotes, otherwise the `:atm:` part is rendered as an emoji (though I guess this depends on whether Pelican renders in the same way, which I doubt it does.)

Reply:
How can I do this? Can you provide an example?

Reply:
Oh, by backquotes I mean the \` character, so you'd need to change the `'urn:xmpp:atm:0'` to `\`urn:xmpp:atm:0\``. Not sure if it's actually a problem on the website itself but it renders as an emoji here on Github (since certain things between colons are rendered as such), so feel free to ignore.

Reply:
Ehm - let us just leave it, I think this is just a Github rendering issue, but not in Pelican... because it cannot :)
Thanks for the hint anyway!